examples/clients/python/seafowl/dataframe_to_sql.py

```diff
@@ -0,0 +1,83 @@
+from contextlib import contextmanager
+from functools import partial
+from typing import Any, Callable, Generator, List
+import pandas as pd
+from sqlalchemy import create_engine
+
+# mypy doesn't recognize that pandas.io.sql exports pandasSQL_builder
+from pandas.io.sql import pandasSQL_builder, SQLTable, PandasSQL  # type:ignore
+import numpy as np
+
+from .seafowl import DEFAULT_DB, DEFAULT_SCHEMA, emit_value, sql_esc_ident, query
+from .types import QualifiedTableName, RowInserter, SeafowlConnectionParams
+
+
+@contextmanager
+def with_pandas_sql() -> Generator[PandasSQL, None, None]:
+    engine = create_engine("sqlite://", echo=True)
+    with pandasSQL_builder(engine, need_transaction=False) as pandas_sql:
+        yield pandas_sql
```

Comment:
What is this magic? 😅  Are we trying to get SQLAlchemy to emit Pandas -> SQL code in the SQLite dialect?

Reply:
Yes, exactly. SQLite and Seafowl have the same type names. The pandas `dtype` to SQL type conversion functions require a specific db adaptor, and we don't have one for Seafowl yet, but the SQLite one works like a charm -at least for generating a `CREATE TABLE ...` statement.